components/activity_and_insights/insights/card/card.scss

```diff
@@ -0,0 +1,75 @@
+.Card.insights-card {
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    margin: 8px;
+    border-radius: 8px;
+    box-shadow: none;
+
+    &.large {
+        width: auto;
+    }
+
+    &.medium {
+        width: calc((100%/2) - 16px);
+        float: left;
+    }
+
+    &.small {
+        width: calc((100%/3) - 16px);
+        float: left;
+    }
+
+    .Card__header {
```

Comment:
in order to make the absolute positioning work (see my comment on the arrow icon button), we will need to add `position: relative` to the card header